.github/actions/build-branch/action.yml

```diff
@@ -6,7 +6,7 @@ inputs:
     required: false
 outputs:
   branch_version_tag:
-    description: 'Tag used for jars and docker images. Either user specified or auto generated as `dev-<commit_hash>`'
+    description: "Tag used for jars and docker images. Either user specified or auto generated as `dev-<commit_hash>`"
```

Comment:
is this for consistency, or was this is actual error?

Reply:
it was pre-commit hook we have in the repo https://github.com/airbytehq/airbyte/blob/master/.pre-commit-config.yaml#L29

Reply:
ty for context